packages/uni_app/lib/session/flows/credentials/request.dart

```diff
@@ -35,11 +37,21 @@ class CredentialsSessionRequest extends SessionRequest {
           'Failed to authenticate user',
           AuthenticationExceptionType.expiredCredentials,
         );
-      } else {
+      } else if (failureReason == LoginFailureReason.internetError) {
+        throw const AuthenticationException(
+          'Failed to authenticate user',
+          AuthenticationExceptionType.internetError,
+        );
+      } else if (failureReason == LoginFailureReason.wrongCredentials) {
         throw const AuthenticationException(
           'Failed to authenticate user',
           AuthenticationExceptionType.wrongCredentials,
         );
+      } else {
+        unawaited(Sentry.captureException(failureReason));
```

Comment:
Don't add sentry captures at this level. Should be the most independent possible. The Sentry captures should be used in the app flow.